### ISSUE TITLE
Partial profile application of Detail settings does not work

### ISSUE THREAD

**Issue by Thanatomanic** [Beep6581/RawTherapee#4945]
To reproduce:
- Enable Sharpening or any other tool on the Detail tab and set to some arbitrary values
- Save the profile
- Apply Neutral
- Apply Partial the previously saved profile, and only select the 'Detail settings' block

Result: the settings do not get applied.
If you fully apply the saved profile, the settings will get applied.

I haven't tested this thoroughly with other settings because of time restraints, but this does not seem to work at least. @Beep6581 Can you confirm?

**Comment by Beep6581** [comment]
@Thanatomanic I just tried to reproduce in `5.4-1098-g`e1742c812 and could not. Can you confirm that your saved PP3 has the `[Sharpening]` section? What was your fill mode when opening the PP3?

**Comment by Thanatomanic** [comment]
> Can you confirm that your saved PP3 has the [Sharpening] section?

Yes, see here: [Sharpening-try.pp3.txt](https://github.com/Beep6581/RawTherapee/files/2560567/Sharpening-try.pp3.txt)

>What was your fill mode when opening the PP3?

Not sure how to call it, but this:
![image](https://user-images.githubusercontent.com/6567747/48181874-77ed3a00-e329-11e8-8615-157dd59e74a4.png)
Changing the mode does not change the outcome for me.

**Comment by Thanatomanic** [comment]
Btw, to reproduce: I use the context menu and use 'Apply - partial'. If I Ctrl+click the 'Load profile from file' button and then select the Detail settings, the settings will be applied as expected.

**Comment by Beep6581** [comment]
Issue confirmed.

1. Open a raw, enable Sharpening, change the settings.
2. Save a PP3 to your "My profiles" folder.
3. Right-click on the thumb in the Filmstrip and Apply Partial your newly saved profile.

When I tick "Sharpening" in the "Partial paste processing profile" window, the Sharpening tool does not get activated and its values don't change.

While testing several times I found that activating the whole "Color Related Settings" section made the Sharpening section load from the PP3 correctly. When I tried each individual checkbox in Color Related Settings (repeat step 3 for each checkbox), none of them made Sharpening apply. When I tried ticking the whole "Color Related Settings" checkbox, now Sharpening did not get applied. 

The same happened with the whole "Basic Settings" checkbox - once it worked, then it didn't.

Maybe some uninitialized value?
